components/data_stores/actions/add-update-multiple-records/add-update-multiple-records.mjs

```diff
@@ -4,8 +4,8 @@ import data_stores from "../../data_stores.app.mjs";
 export default {
   key: "data_stores-add-update-multiple-records",
   name: "Add or update multiple records",
-  description: "Add or update multiple records to your [Pipedream Data Store](https://pipedream.com/data-stores/).",
```

Comment:
actually this description should stay the same

Reply:
Sorry I accidentally changed the wrong description

Reply:
@feyzullah it looks like you did not revert this change. This action description should be,
```
Add or update multiple records to your [Pipedream Data Store](https://pipedream.com/data-stores/).
```